### ISSUE TITLE
Minimize & Maxmize

### ISSUE THREAD

**Issue by mcjoeng** [horst3180/Vertex-theme#134]
Look at this

![2015-09-23 16 43 08](https://cloud.githubusercontent.com/assets/10485304/10041055/e67e8ac2-6213-11e5-9948-62d5f050d85e.png)

**Comment by mcjoeng** [comment]
![2015-09-23 17 24 34](https://cloud.githubusercontent.com/assets/10485304/10041630/fdbceeaa-6217-11e5-9fe5-8b0f530ccae3.png)

**Comment by diogoleitao** [comment]
Are you sure it's not just showing the maximize button only? Just asking to make sure. Either way, I don't have that problem on the latest version (using all three buttons: minimize, maximize and close).

**Comment by mcjoeng** [comment]
gnome-control-center shows only two buttuns,you can run it to check it out

**Comment by diogoleitao** [comment]
Right, I forgot some windows don't have the minimize button. I'll check on my gnome to see if I have the same problem

**Comment by horst3180** [comment]
It's a limitation of GTK. I use a hack to get custom styled window buttons and unfortunately they get messed up on some button layouts.
There's nothing I can do about it other than completely drop the window button styling. FWIW, it will work properly with Gnome 3.16 and newer.
